config/map_layers.yml

```diff
@@ -158,4 +158,4 @@ open_map_surfer:
     opacity: 100
   variants:
     admin_bounds:
-      url: 'http://korona.geog.uni-heidelberg.de/tiles/adminb/x={x}&y={y}&z={z}'
\ No newline at end of file
+      url: 'http://korona.geog.uni-heidelberg.de/tiles/adminb/x={x}&y={y}&z={z}'
```

Comment:
Can you revert this change? Hint: `git checkout -f master -- config/map_layers.yml`